src/firebase.config.ts

```diff
@@ -9,12 +9,13 @@ import { getAnalytics } from "firebase/analytics";
 
 // Your web app's Firebase configuration
 const firebaseConfig = {
-  apiKey: process.env.REACT_APP_API_KEY,
-  authDomain: process.env.REACT_APP_AUTHDOMAIN,
-  projectId: process.env.REACT_APP_PROJECT_ID,
-  storageBucket: process.env.REACT_APP_STORAGE_BUCKET,
-  messagingSenderId: process.env.REACT_APP_MESSAGE_SENDER_ID,
-  appId: process.env.REACT_APP_APP_ID,
+    apiKey: "AIzaSyDXd9yNwLvXvWQ3iWOeIPCEWVhaYRVF3SE",
+    authDomain: "wtfuck-d0de5.firebaseapp.com",
+    databaseURL: "https://wtfuck-d0de5-default-rtdb.firebaseio.com",
+    projectId: "wtfuck-d0de5",
+    storageBucket: "wtfuck-d0de5.appspot.com",
+    messagingSenderId: "314438364070",
+    appId: "1:314438364070:web:e9cc9d978a84a5446abd67"
```

Comment:
```suggestion
    apiKey: "AIzaSyDXd9yNwLvXvWQ3iWOeIPCEWVhaYRVF3SE",
    authDomain: "wtfuck-d0de5.firebaseapp.com",
    databaseURL: "https://wtfuck-d0de5-default-rtdb.firebaseio.com",
    projectId: "wtfuck-d0de5",
    storageBucket: "wtfuck-d0de5.appspot.com",
    messagingSenderId: "314438364070",
    appId: "1:314438364070:web:e9cc9d978a84a5446abd67"
```

use .env.local
I'll put admin id in .env.local in my next commit